benchmarks/benchmarks/stats.py

```diff
@@ -353,3 +353,19 @@ def time_skew(self, order, size, bias):
 
     def time_kurtosis(self, order, size, bias):
         stats.kurtosis(self.x, bias=bias)
+
+
+class BenchQMCDiscrepancy(Benchmark):
+    param_names = ['iterative', 'method']
```

Comment:
Don't really need to test for iterative as this is just changing `n`. The rest of the code being the same.

Reply:
> Don't really need to test for iterative as this is just changing `n`. The rest of the code being the same.

Removed it in 7dcf9cc7d4bbaa0cda4f02fecc6f4e8b43152bd9